v2/server/test/articles.js

```diff
@@ -0,0 +1,123 @@
+const articleToShare = {
```

Comment:
why do we have article tests in an Auth feature PR?

Reply:
These are tests working with the Data Structures **(V1)** I kept it to avoid the Test Coverage to decrease a lot as the changes progress since some mates face problem with their LAF